requirements.txt

```
@@ -1,3 +1,3 @@
 general-superstaq~=0.3.7
 matplotlib~=3.0
-qiskit>=0.35.0, <0.37.0
+qiskit>=0.37.0
```

this will probably break when terra 0.23.0 is released. wdyt about
```suggestion
qiskit>=0.37.0
qiskit-terra==0.22.*
```
(just pinning qiskit~=0.37.0 would also work but it's possible it's stricter than necessary because qiskit 0.38.0 might still use terra 0.22.3)

---

Done.